server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -319,9 +319,9 @@ public void delete(SortedMap<String,String> confirmedDeletes) throws TableNotFou
               // of deleting something that should not be deleted. Must not change value of delete
               // variable because thats whats stored in metadata table.
               log.debug("Volume replaced {} -> {}", delete, switchedDelete);
-              fullPath = fs.getFullPath(FileType.TABLE, switchedDelete);
+              fullPath = new Path(switchedDelete);
```

Comment:
Should these be validated?

Reply:
Looking at the code, I can't figure out if these are delete markers or the files to delete...

Reply:
These are delete markers that have no references and are therefore safe to delete.  The code after this deletes the files and then deletes the markers from the metadata table.